convert/src/vertex_to_bd/badgerdoc_format/annotation_converter.py

```diff
@@ -0,0 +1,118 @@
+from typing import Any, List, Optional, Tuple
+
+from ..models import LabelStudioModel, ModelItem, ResultItem
+from ..models.bd_annotation_model import BadgerdocAnnotation, Obj, Page, Size
+from ..models.bd_tokens_model import Page as BadgerdocTokensPage
+
+LABELS = "labels"
+RELATION = "relation"
```

Comment:
since you use it in this class, may you move them directly there? Or I miss something?

Reply:
You are right. Moved